bc-wikitemplate.md

```diff
@@ -1,87 +1,103 @@
 
-## Installer
+
+### Crash verification
+- [ ] Verify these [crash issues](https://github.com/brave/brave-browser/issues?utf8=%E2%9C%93&q=is%3Aissue+milestone%3A%22Releasable+builds+0.55.x%22+label%3Acrash+is%3Aclosed++label%3AQA%2FNeeded) have not regressed 
+- [ ] Verify these [webview crash issues](https://github.com/brave/brave-browser/issues?utf8=%E2%9C%93&q=is%3Aissue+milestone%3A%22Releasable+builds+0.55.x%22+label%3Acrash/webview+is%3Aclosed++label%3AQA%2FNeeded) have not regressed 
+
+
+### Installer
 
 - [ ] Check that installer is close to the size of last release.
 - [ ] Check signature: If OS Run `spctl --assess --verbose /Applications/Brave-Browser.app/` and make sure it returns `accepted`.  If Windows right click on the installer exe and go to Properties, go to the Digital Signatures tab and double click on the signature.  Make sure it says "The digital signature is OK" in the popup window.
```

Comment:
I know the current QA team will know that the `spctl --assess --verbose /Applications/Brave-Browser.app/` might differ while on different channels but we should explicitly mention which locations need to be checked depending on the channel. Example: `spctl --assess --verbose /Applications/Brave-Browser-Dev.app/`

@bbondy @srirambv thoughts?

We should also find out a way if there's a way of checking the signatures on Linux (at least Ubunutu).